cg/models/scout/scout_load_config.py

```diff
@@ -177,6 +178,7 @@ class NalloLoadConfig(ScoutLoadConfig):
     peddy_check: str | None = None
     peddy_ped: str | None = None
     peddy_sex: str | None = None
+    reviewer: Reviewer = Reviewer(trgt="true")
```

Comment:
Hm, no, this is sample level information - compare L103/104 above in the ScoutRarediseaseIndividual!